src/pybind/mgr/mgr_module.py

```diff
@@ -520,21 +529,21 @@ def __init__(self, module_inst):
 
     def emit(self, record):
         levelmap = {
-            'DEBUG': MgrModule.CLUSTER_LOG_PRIO_DEBUG,
-            'INFO': MgrModule.CLUSTER_LOG_PRIO_INFO,
-            'WARNING': MgrModule.CLUSTER_LOG_PRIO_WARN,
-            'ERROR': MgrModule.CLUSTER_LOG_PRIO_ERROR,
-            'CRITICAL': MgrModule.CLUSTER_LOG_PRIO_ERROR,
+            logging.DEBUG: MgrModule.ClusterLogPrio.DEBUG,
+            logging.INFO: MgrModule.ClusterLogPrio.INFO,
+            logging.WARNING: MgrModule.ClusterLogPrio.WARN,
+            logging.ERROR: MgrModule.ClusterLogPrio.ERROR,
+            logging.CRITICAL: MgrModule.ClusterLogPrio.ERROR,
         }
-        level = levelmap[record.levelname]
+        level = levelmap[record.levelno]
         if record.levelno >= self.level:
             self._module.cluster_log(self._module.module_name,
                                      level,
                                      self.format(record))
 
 
 class FileHandler(logging.FileHandler):
-    def __init__(self, module_inst):
+    def __init__(self, module_inst: Any):
```

Comment:
There is a way to avoid Any here:

```python
Module_T = TypeVar('Module_T', bound="MgrModule")

class FileHandler(logging.FileHandler, Generic[Module_T]):
    def __init__(self, module_inst: Module_T):
```

But I doubt it's worth it here.

Reply:
seems the suggest change fail to parse / run. I haven't looked into the failure. am using `Any` for now. will revisit this part once this change gets merged.